css/index.css

```diff
@@ -0,0 +1,247 @@
+html {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  body {
+    background: #f8f8f8;
+    color: #303438;
+    font-family: "Lato","Helvetica Neue",Helvetica,Arial,sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+  svg {
+    fill: currentColor;
+  }
+  
+  a {
+    color: #1F0C7B;
+    text-decoration: none;
+  }
+    a:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  
+  h1 {
+    text-align: center;
+    margin-bottom: 1.5em;
+    font-size: 2.5em;
+    font-family: "Merriweather",serif;
+  }
+  div + h1 {
+    margin-top: 3em;
+  }
+  @media screen and (max-width: 600px) {
+    h1 {
+      font-size: 2.1em;
+    }
+  }
+  
+  .header {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: black;
+    color: #f8f8f8;
+  }
+    .header > a {
+      margin: .8em;
+      padding: 0.7em;
+      color: white;
+      display: inline-block;
+      font-size: 1.1em;
+    }
+    .header > p:last-child {
+      margin-right: 2em;
+    }
+    .header > a:hover {
+      text-decoration: none;
+    }
+    .header img {
+      padding-right: 0.8em;
+      vertical-align: middle;
+    }
+  @media screen and (max-width: 600px) {
+    .header > p:last-child {
+      display: none;
+    }
+  }
+  button {
+    float: right;
+    background: none;
+    border: none;
+    padding: 0 4px;
+    font-size: 100%;
+    cursor: pointer;
+    transition: opacity 0.3s ease-out;
+  }
+    button:focus {
+      outline: 1px dotted;
+    }
+    button:hover {
+      opacity: 0.7;
+    }
+
+    input[type="text"]:focus {
+      outline: none;
+      border: solid 5px #969696;
+    }
+  @media screen and (max-width: 1200px) {
+    .instructions {
+      width: 80%;
+    }
+  }
+  @media screen and (max-width: 600px) {
+    .instructions {
+      width: 92%;
+    }
+  }
+  .instructions {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
+    overflow: hidden;
+    width: 65%;
+    max-width: 1300px;
+  }
+  
+  input[type="file"] {
+      width: 0.1px;
+      height: 0.1px;
```

Comment:
[dust] What's going on here?

Reply:
Digging into how to restructure this helped me understand exactly why this was done. The input field to select the input file has ugly default styling that is difficult to remove: 
<img width="477" alt="image" src="https://github.com/newjersey/taxation-mef-viewer/assets/4315803/374970de-3b23-492e-9e8e-547c778fd6fd">

By setting width/height to 0.1px, z-index to -1, and opacity to 0, we maintain the functionality of allowing users to select files by clicking on the easily styled label, but prevent them from seeing the input field.

> This is a native part of the webkit browsers and you cannot remove it. You should think about a hacky solution like covering or hiding the file inputs.

https://stackoverflow.com/questions/12035400/how-can-i-remove-the-no-file-chosen-tooltip-from-a-file-input-in-chrome